### DISCUSSION TITLE
Indent Tool is missing

### DISCUSSION THREAD

**Discussion by sahudhirendra** [sibiraj-s/ngx-editor#407] [Q&A] [answered]
Is this editor supports increase and decrease indent?

**Comment by marchrenno** [comment]
It depends what u mean by indent. 
It currently supports embedding ol and ul lists within each other... which creates an indentation.
However, if by indentation you mean purely graphical indentation, then you will have to create a node and add it to the schema to allow this, and then add the buttons in the toolbar.

**Reply by sibiraj-s** [reply]
Thanks @marchrenno .

**Comment by sibiraj-s** [comment]
@sahudhirendra. Any custom options you could add a schema for it and define the toolbar icons using custom menu option.

For writing schemas, See https://prosemirror.net/examples/schema/

Happy to accept a PR.

**Reply by sibiraj-s** [reply, marked as answer]
indent/outdent is now supported in v17+

[a comment by an automated account is not shown]
